lib/facebook.ex

```diff
@@ -83,6 +83,30 @@ defmodule Facebook do
   end
 
   @doc """
+  A Picture for a Facebook User
+
+  See: https://developers.facebook.com/docs/graph-api/reference/user/picture/
+  """
+  @spec picture(type :: String.t, access_token) :: response
+  def picture(type, access_token) do
+    picture(type, access_token, [])
+  end
+
+  @doc """
+  A Picture for a Facebook User
+
+  See: https://developers.facebook.com/docs/graph-api/reference/user/picture/
+  """
+  @spec picture(type :: String.t, access_token, options) :: response
+  def picture(type, access_token, options) do
+    fields = if !is_nil(Config.appsecret) do
+      [type: type, redirect: false, appsecret_proof: encrypt(access_token)]
```

Comment:
Ok. I see. But does this actually work with the conditional without a default for the `fields` variable?

e.g.
```
» iex
Erlang/OTP 18 [erts-7.3] [source] [64-bit] [smp:4:4] [async-threads:10] [hipe] [kernel-poll:false] [dtrace]

Interactive Elixir (1.0.4) - press Ctrl+C to exit (type h() ENTER for help)
iex(1)> foo = if false do
...(1)> [test: "this"]
...(1)> end
nil
iex(2)> foo ++ [doesit: "work?"]
** (ArgumentError) argument error
    :erlang.++(nil, [doesit: "work?"])
```

In the other cases fields is defined already as an argument to the function..